eclipse-collections-api/src/main/java/org/eclipse/collections/api/InternalIterable.java

```diff
@@ -49,6 +51,13 @@
     @SuppressWarnings("UnnecessaryFullyQualifiedName")
     void forEach(Procedure<? super T> procedure);
 
+    @Override
+    default void forEach(Consumer<? super T> consumer)
+    {
+        Procedure<? super T> procedure = consumer::accept;
+        forEach(procedure);
```

Comment:
To keep in line with our convention please change this to `this.forEach(procedure)`